rest-server/src/routes/job.js

```diff
@@ -40,6 +40,9 @@ router.route('/:jobName')
     /** DELETE /api/v1/jobs/:jobName - Remove job */
     .delete(tokenConfig.check, jobController.remove);
 
+router.route('/:jobName/executionType')
```

Comment:
Let's follow the design of WebHDFS (see https://hadoop.apache.org/docs/r1.0.4/webhdfs.html#CREATE):
- To create a job: `PUT /:jobName?op=UPDATE` (with data).
- To stop a job: `PUT /:jobName?op=STOP` (without data).

Reply:
Why follow it?

---
In reply to: [173090250](https://github.com/Microsoft/pai/pull/375#discussion_r173090250) [](ancestors = 173090250)

Reply:
Good practice. Current design is weird because ‘executionType’ is NOT a resource.